### PR TITLE
Feature/ login

### PR DESCRIPTION
**Pull request by charlesmuchene**

### CODE REVIEW

File: api/controllers/user.controller.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+exports.create=(request,response)=>{
+    const user=new User();
+    user.fullName=request.body.fullName,
+    user.password=request.body.password,
+    user.email=request.body.email,
+    user.save().then((data) => response.json(data)).catch((error) =>
+    response.status(500).json({
+        error: {
+            code: 500,
+            message: error.message || 'Error while creating a session'
```
Comment (charlesmuchene) [inline api/controllers/user.controller.js]: Check the error message. It should say user.

File: api/controllers/user.controller.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const User=require('../models/user.model');
```
Comment (charlesmuchene) [inline api/controllers/user.controller.js:1]: Do you need to implement other methods? Update, Delete?

File: api/controllers/user.controller.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+//Create and save a new user
+exports.create=(request,response)=>{
+    const user=new User();
+    user.fullName=request.body.fullName,
+    user.password=request.body.password,
```
Comment (charlesmuchene) [inline api/controllers/user.controller.js:7]: I don't see confirm password. Typically you let the user type the password twice.

File: api/routes/user.routes.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+const router=require('express').Router();
+const user=require('../controllers/user.controller.js');
+
+//Retrive all users
+router.get('/',user.findAll);
+
+//Retrieve user with the given id
+//router.get('/:userId',user.findOne);
```
Comment (charlesmuchene) [inline api/routes/user.routes.js]: Activate this route.

File: client/src/app/login/login.component.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      ]],
+      'password': [this.user.password, [
+        Validators.required,
+        Validators.minLength(6),
+        Validators.maxLength(30)
+      ]]
+    });
+  }
+  onLoginSubmit(){
+    alert(this.user.email + ''+ this.user.password);
```
Comment (charlesmuchene) [inline client/src/app/login/login.component.ts:34]: Great! Next you can now post this to the api.

**Comment by RedaeMengsteab** [comment]
good feedback

File: api/controllers/user.controller.js (hunk 2 above)
Comment (RedaeMengsteab) [reply api/controllers/user.controller.js:1]: yes, i have already added them

**Pull request closed without merging**